graalwasm/graalwasm-embed-c-code-guide/README.md

```diff
@@ -0,0 +1,196 @@
+# Embed C in Java Using GraalWasm
+
+The example below demonstrates how to compile a C function to WebAssembly and run it embedded in a Java application.
+
+## 1. Getting Started
```

Comment:
This is not a "getting started" section, but "prerequisites".

Reply:
It is not a step 1. There are no steps to complete. I would change the title to `### Prerequisites`  and start the next section as step 1.